docker-compose-setup/.env

```diff
@@ -0,0 +1,14 @@
+TAG=2.0.0-SNAPSHOT
```

Comment:
This is actually the coatrack version from the pom, but that version in the pom can change. So I was wondering if the script can 
- either actually take the version number from the pom
- or use a more generic TAG like "local-snapshot-build"?

Reply:
Both approaches seem to be easy to implement. The background is that when 'mvn package' is executed a proxy jar file with some name like 'coatrack-proxy-<TAG>.jar' is generated. So the tag is meant to find that file. I am not certain yet which approach would be better.